app/models/main.py

```diff
@@ -1319,6 +1320,16 @@ class Brief(db.Model):
         viewonly=True
     )
 
+    process_outcome = db.relationship(
```

Comment:
(So I'm proposing that this should not be in this PR either. - sorry this is my bad as obviously I wasn't clear at the design stage, that I was trying to get us to design something _suitable_ for Briefs, but I _wasn't_ trying to get us to implement anything at this stage.)

Reply:
I think leaving the design open "so the possibility exists" to also work with briefs is as good as not considering it in the first place. Whoever came to deal with that later would have their own ideas.

Reply:
This is an argument against doing any architecture at all, or even any technical roadmapping. I can't buy that argument, because it means everything has to be short-term. We know in this case that we _are_ going to need it, we just don't need it _yet_, and the question is just about what gets implemented as code _at this stage_, surely?